ironfish/src/mining/webhooks/webhookNotifier.ts

```diff
@@ -78,6 +79,31 @@ export abstract class WebhookNotifier {
     )
   }
 
+  poolPayoutStarted(
+    payoutId: number,
+    receives: { publicAddress: string; amount: string; memo: string }[],
+    totalShareCount: number,
+  ): void {
+    const total = receives.reduce((m, c) => BigInt(c.amount) + m, BigInt(0))
+
+    this.sendText(
+      `Creating payout of ${totalShareCount} shares to ${
+        receives.length
+      } users for ${displayIronAmountWithCurrency(
+        Number(oreToIron(Number(total.toString()))),
+        false,
+      )}(${payoutId})`,
+    )
+  }
+
+  poolStatus(status: MiningPoolStatus): void {
+    this.sendText(
+      `Status for mining pool '${status.name}':\n\tHashrate: ${FileUtils.formatHashRate(
+        status.hashRate,
+      )}/s\n\tMiners: ${status.miners}\n\tShares pending: ${status.sharesPending}`,
+    )
+  }
+
```

Comment:
Hopefully this can be deleted later in favor of a poll mechanic so it doesn't spam notifications.